extensions/amp-lightbox-gallery/1.0/provider.js

```diff
@@ -149,3 +185,70 @@ function NavButtonIcon({by, ...rest}) {
     </svg>
   );
 }
+
+/**
+ * @param {!BaseCarouselDef.ArrowProps} props
+ * @return {PreactDef.Renderable}
+ */
+function ToggleViewIcon({showCarousel, ...rest}) {
+  const classes = useStyles();
+  return (
+    <svg
+      aria-label={`Switch to ${showCarousel ? 'grid view' : 'carousel view'}`}
```

Comment:
Probably not a concern now, but if this string ever needs to be translated this will not work. Better to do
`aria-label={showCarousel ? 'Switch to grid view' : 'Switch to carousel view'}` to future proof this.

Reply:
Good idea, updated. Thanks!